src/util/formDevices.tsx

```diff
@@ -98,11 +98,25 @@ export const formDeviceToPayload = (devices: FormDevice[]) => {
       if ("size" in item && !item.size?.match(/^\d/)) {
         delete item.size;
       }
+      if (
+        (item as LxdNicDevice) &&
+        (item as LxdNicDevice)["security.acls"] !== undefined
+      ) {
+        return {
+          ...obj,
+          [name]: {
+            ...item,
+            "security.acls": (item as LxdNicDevice)["security.acls"],
+          },
+        };
+      }
```

Comment:
```suggestion
```
This should not be needed, I think this function can stay as it was. I briefly tested commenting this out and it worked for me on editing an instance also without.